### DISCUSSION TITLE
Question/request: Move single website from one dashboard to another

### DISCUSSION THREAD

**Discussion by gridsystem** [umami-software/umami#1822] [Ideas]
Apologies if this isn't the right place for this.

I have an umami dashboard with multiple websites.

If I want to move a single site out of my dashboard and into it's own dedicated dashboard, is it possible to export from the original dashboard and then import to the new one, either through the UI, by running a script, or getting stuck into SQL?

This feels like something I'm not the first person to think about, but can't see anything in the repo discussions about this use case, so thought I'd ask.

btw - I love this software!

**Comment by tschannik** [comment]
I can only agree! Would be an awesome feature (but probably not a huge usecase)

**Comment by mikecao** [comment]
Do you mean custom dashboards? Because that is a planned future.

For now, you can create another account, then assign that website to the new account (via SQL). Then you can visit that dashboard as admin.

**Comment by gridsystem** [comment]
Not entirely, I’m talking about two separate instances of Umami - A and B.

A has 10 sites.
B has no sites.

A exports one of it’s sites and imports it into B. A then deletes that site.

A has 9 sites.
B has 1 site which contains all the analytics data from when it existed in Umami A.

User changes Umami URL in html to stop pointing at Umami A and to now point at Umami B.

**Comment by mikecao** [comment]
Probably the easiest thing to do is to make a copy of the database, then delete all the websites you don't want from the copy.

**Comment by gridsystem** [comment]
Sounds like a plan, thanks for that :) No need to overcomplicate things if it works.

**Comment by bbilly1** [comment]
I would also appreciate the flexibility to move websites between installations. Copying the database might not always be an option if you already have other websites stored on that host.

For example when you have:
- Host 1:
  - Website 1
  - Website 2
- Host 2:
  - Website 3
  - Website 4

And you want to move **Website 3** to **Host 1** but you already have data for **Website 1** and **Website 2** that you care about. So more of a *merge* than a copy.

Maybe a per Website backup? Then a restore that could add that as a new site.

**Comment by franciscao633** [comment]
Moving to ideas for the time being. Probably won't be a priority to implement this type of workflow for its current case, Could possibly be added as part of imports / exports in the future.
